api/test/docker/manager-api-conf.yaml

```diff
@@ -25,6 +25,8 @@
       - 172.16.238.10:2379   # ips here are defined in docker compose.
       - 172.16.238.11:2379
       - 172.16.238.12:2379
+  online_debug:
+     enable: true
```

Comment:
Default false

Reply:
It's test/docker/manager-api-conf.We should enable it to test online debug.

Reply:
ok